modules/dlm/src/test/java/org/elasticsearch/dlm/action/ExplainDataLifecycleResponseTests.java

```diff
@@ -186,6 +205,7 @@ private static ExplainIndexDataLifecycle createRandomIndexDLMExplanation(long no
             true,
             now,
             randomBoolean() ? now + TimeValue.timeValueDays(1).getMillis() : null,
+            TimeValue.timeValueMillis(now),
```

Comment:
Sounds good. I'll leave in the explicit test of null behavior at the end of testToXContent() though so that we don't have to depend on randomness to detect any regressions.